### PR TITLE
#157803964 Add side menu

### PR DESCRIPTION
**Pull request by Arthur236**
## What does this PR do?
* Add a side menu for navigation links
* Add a navigation bar

## Description of Task to be completed?
* Implement a side menu to hold links to specific pages
* Implement a navigation bar to show page title and the username of the logged in user

## How should this be manually tested?
* N/A

## What are the relevant pivotal tracker stories?
[157803964](https://www.pivotaltracker.com/story/show/157803964)

## Any background context you want to add?
N/A

## Important notes
* Any component that is intended to have the side menu should be wrapped between `<SideMenuComponent></SideMenuComponent>`

## Packages installed
* jsonwebtoken

## Deployment note
N/A

## Screenshots
screen shot 2018-05-30 at 15 47 10

screen shot 2018-05-30 at 15 47 25

### CODE REVIEW

File: src/__test__/SideMenuComponent.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import expect from 'expect';
+
+import SideMenu from '../_components/SideMenuComponent';
+
+describe('Renders <SideMenu /> correctly', () => {
```
[the remark this answers is not in the document]
Comment (Arthur236) [reply src/__test__/SideMenuComponent.test.js]: Noted

File: src/_components/NavBarComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import jwt from 'jsonwebtoken';
+import { Dropdown, Menu, Icon } from 'semantic-ui-react';
+
+import '../_css/NavBarComponent.css';
+
+const NavBarComponent = (props) => {
```
Comment (sachungo) [inline src/_components/NavBarComponent.jsx]: - Select everything inside `NavBarComponent` and move back by 2 spaces. If you are using visual studio, then the command to use after selecting is `cmd + [`. 
- You should also set your editor to use **_2 spaces_** for js/jsx files

File: src/_components/SideMenuComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  toggleVisibility = () => this.setState({ visible: !this.state.visible });
+
+  render() {
+    const { visible } = this.state;
+    const { title } = this.props;
+
+    return (
+      <div className='wrapper'>
+        <Sidebar.Pushable as={ Segment }>
```
Comment (sachungo) [inline src/_components/SideMenuComponent.jsx]: Only destructured objects  should have the space after `{` or before `}`. Values passed to elements shouldn't have the spaces. So this should just be `as={Segment}`. I'll talk to Eric about integrating prettier to the codebase so as not to worry about the nitty-gritty

Comment (Arthur236) [reply src/_components/SideMenuComponent.jsx]: Awesome. Let me rectify that.

**Pull request merged**